### PR TITLE
Update to swift3.0

### PR DESCRIPTION
**Pull request by iceman201**
Notes: there is one comments which Im not too sure how to re-write, please check out around _line 143-144_

### CODE REVIEW

File: SwiftNotice.swift [hunk 1]
```diff
@@ -137,20 +137,22 @@ class SwiftNotice: NSObject {
...
                 let iv = UIImageView(frame: frame)
                 iv.image = imageNames.first!
-                iv.contentMode = UIViewContentMode.ScaleAspectFit
+                iv.contentMode = UIViewContentMode.scaleAspectFit
                 mainView.addSubview(iv)
+                /*
                 timer = dispatch_source_create(DISPATCH_SOURCE_TYPE_TIMER, 0, 0, dispatch_get_main_queue())
-                dispatch_source_set_timer(timer, DISPATCH_TIME_NOW, UInt64(timeInterval) * NSEC_PER_MSEC, 0)
-                dispatch_source_set_event_handler(timer, { () -> Void in
+                (timer as! DispatchSource).setTimer(start: dispatch_time_t(DISPATCH_TIME_NOW), interval: UInt64(timeInterval) * NSEC_PER_MSEC, leeway: 0)
```
Comment (iceman201) [inline SwiftNotice.swift:144]: About here

**Review by iceman201 — COMMENTED** [review]
not too sure how to re-write that part

File: SwiftNotice.swift (hunk 1 above)
Comment (johnlui) [reply SwiftNotice.swift:144]: haha, I already has a fix.

**Comment by johnlui** [comment]
Thanks for your pr, and I will do the rest work for you.

**Pull request merged**